xbmc/video/ContextMenus.cpp

```diff
@@ -168,4 +171,26 @@ bool CPlay::Execute(const CFileItemPtr& itemIn) const
   return true;
 };
 
+
+std::string CFavorite::GetLabel(const CFileItem& item) const
+{
+  //The 0 we're passing for contextWindow should only matter to folders
+  return g_localizeStrings.Get(XFILE::CFavouritesDirectory::IsFavourite(std::make_shared<CFileItem>(item).get(), 0) ? 14077 : 14076); // Add/Remove Favourite
```

Comment:
@tamland so anything I can do about it or can we merge it as is, as it basically should be the same behaviour as before